Jenkinsfile

```diff
@@ -0,0 +1,142 @@
+@Library('socrata-pipeline-library')
+
+def deploy_environment = "staging"
+def service_sha = env.GIT_COMMIT
+
+// Stage control variables to simplify stage selection by build cause
+def stage_cut = false
+def stage_build = false
+def stage_dockerize = false
+def stage_deploy = false
+
+// Utility Libraries
+def sbtbuild = new com.socrata.SBTBuild(steps, "query-coordinator", "query-coordinator")
+def dockerize = new com.socrata.Dockerize(steps, "query-coordinator", BUILD_NUMBER)
+def deploy = new com.socrata.MarathonDeploy(steps)
+
+pipeline {
+  options {
+    ansiColor('xterm')
+  }
+  agent {
```

Comment:
you can make this a parameter on the job like so : 
    parameters {
        string(name: 'AGENT', defaultValue: 'worker-pg9x-bionic-chef14', description: 'Which build agent to use?')
    }
agent { label params.AGENT }

Reply:
Would we ever want it parameterized since running on multi-executor workers (or master) opens it up sbt builds to ivy2 conflicts?  (Unless we're really diligent about preventing those by job families)

Reply:
Added params, but that didn't get picked up for this conversation by github, I guess...